external_resources/README.md

```diff
@@ -1,48 +1,162 @@
 
 **SECTIONS**
 
 1. [Overview](#overview)
-1. [Enabling Task](#enabling-task)
+1. [Enabling Tasks](#enabling-tasks)
+1. [Wayback Machine Integration](#wayback-machine-integration-1)
+1. [Wayback Machine Removal Requests](#wayback-machine-removal-requests)
 1. [Frequency Control](#frequency-control)
 1. [Rate Limiting](#rate-limiting)
 1. [Task Priority](#task-priority)
-
+1. [Management Commands](#management-commands)
+1. [Code References](#code-references)
 
 # Overview
 
-This assumes that celery beat scheduler is installed and enabled, which is required for the task scheduling.
+This assumes that **Celery beat scheduler** is installed and enabled, which is required for the task scheduling.
+
+The frequency for **external resource checking** is set to once per week (`1/week`). All external resources, both new and existing, are validated weekly, regardless of their last status.
+
+The **Wayback Machine integration** is enabled by default. Valid external resources are submitted for archiving as part of the external resource validation task. The system monitors the submission status and avoids resubmitting resources within a specified interval. The status of Wayback Machine archiving jobs is updated every 6 hours to track the progress of pending jobs.
+
+If a resource is valid, the `check_external_resources` task triggers the `submit_url_to_wayback_task` to archive it.
+
+Here’s a high-level description of the process:
+
+### External Resource Validation:
+
+- Task is automatically added in scheduler on system start.
+- On execution, all available external resources are retrieved from DB.
+- Gathered data is divided into preconfigured batch sizes, defined by `BATCH_SIZE_EXTERNAL_RESOURCE_STATUS_CHECK` in [websites/constants.py](/websites/constants.py), which is currently set to `100`.
+- All batches are grouped into a single Celery task and executed.
+- Each batch-task iterates over batch to validate availability of each resource and its backup resource if available.
+- The status of resource is then added to DB.
+- Batch tasks have a preconfigured rate limiter and lower priority by default.
+
+### Wayback Machine Integration:
+
+- When external resource validation occurs, valid external resources are submitted to the Wayback Machine for archiving.
+- The status of Wayback Machine archiving jobs is tracked and updated periodically (currently, every 6 hours).
+- New external resources are automatically submitted to the Wayback Machine upon creation.
+
+# Enabling Tasks
+
+### External Resource Checking:
+
+The task for external resource checking can be enabled/disabled using the `CHECK_EXTERNAL_RESOURCE_TASK_ENABLE` defined in [settings.py](/main/settings.py). By default, this task is enabled.
+
+### Wayback Machine Tasks:
+
+Wayback Machine tasks can be enabled or disabled using the `ENABLE_WAYBACK_TASKS` defined in [settings.py](/main/settings.py). These tasks are also enabled by default.
```

Comment:
Maybe too late for this comment, but I wish the naming scheme for `CHECK_EXTERNAL_RESOURCE_TASK_ENABLE` and `ENABLE_WAYBACK_TASKS` were more similar. 

i.e. 
`CHECK_EXTERNAL_RESOURCE_TASK_ENABLE` and `WAYBACK_TASKS_ENABLE`

or 
`ENABLE_CHECK_EXTERNAL_RESOURCE_TASK` and `ENABLE_WAYBACK_TASKS`

Reply:
I'm going to change `CHECK_EXTERNAL_RESOURCE_TASK_ENABLE` into `ENABLE_CHECK_EXTERNAL_RESOURCE_TASK`